lmu2png/src/main.cpp

```diff
@@ -145,15 +160,20 @@ void DrawEvents(SDL_Surface* output_img, stChipset * gen, std::unique_ptr<lcf::r
 			}
 
 			SDL_Surface* charset_img(LoadImage(charset.c_str(), true));
-			SDL_Rect src_rect {(evp->character_index % 4) * 72 + evp->character_pattern * 24,
-				 (evp->character_index / 4) * 128 + evp->character_direction * 32, 24, 32};
+			SDL_Rect src_rect;
+			if (!simulate_movement || (simulate_movement && evp->animation_type))
```

Comment:
This condition makes no sense to me, can you explain?

Reply:
simulate movement replicates a common trick where events set to the "standing animation" anim type are given a charset with editor-only icons on the sides and an empty row in the middle. when the game runs, it updates the event to show the middle row instead of the side rows. this condition ensures that the simulate movement option is only applied to standing animation anim type events